internal/metadata/metadata.go

```diff
@@ -11,6 +11,7 @@ import (
 
 	"github.com/xmidt-org/wrp-go/v3"
 	"github.com/xmidt-org/xmidt-agent/internal/net"
+	b64 "encoding/base64"
```

Comment:
unless `base64` is conflicting with something I'd use that, otherwise this looks good.